server/graphql/v2/mutation/MemberInvitationMutations.js

```diff
@@ -0,0 +1,151 @@
+        throw new Unauthorized('Only admins can send an invitation.');
+      }
+
+      if (![MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(args.role)) {
+        throw new Forbidden('You can only invite accountants, admins, or members.');
+      }
+
+      const memberParams = {
+        ...pick(args, ['role', 'description', 'since']),
+        MemberCollectiveId: memberAccount.id,
+        CreatedByUserId: req.remoteUser.id,
+      };
+
+      // Invite member
+      return models.MemberInvitation.invite(account, memberParams);
+    },
+  },
+  editMemberInvitation: {
+    type: MemberInvitation,
+    description: 'Edit an existing member invitation of the Collective',
+    args: {
+      memberAccount: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to an account for the member to edit.',
+      },
+      account: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to an account for the Collective',
+      },
+      role: {
+        type: MemberRole,
+        description: 'Role of member',
+      },
+      description: {
+        type: GraphQLString,
+      },
+      since: {
+        type: ISODateTime,
+      },
+    },
+    async resolve(_, args, req) {
+      if (!req.remoteUser) {
+        throw new Unauthorized('You need to be logged in to invite a member.');
+      }
+
+      let { memberAccount, account } = args;
+
+      memberAccount = await fetchAccountWithReference(memberAccount, { throwIfMissing: true });
+      account = await fetchAccountWithReference(account, { throwIfMissing: true });
+
+      if (!req.remoteUser.isAdminOfCollective(account)) {
+        throw new Unauthorized('Only admins can edit members.');
+      }
+
+      if (![MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(args.role)) {
+        throw new Forbidden('You can only edit accountants, admins, or members.');
+      }
+
+      // Edit member invitation
+      const editableAttributes = pick(args, ['role', 'description', 'since']);
+
+      return models.MemberInvitation.update(editableAttributes, {
+        where: {
+          MemberCollectiveId: memberAccount.id,
+          CollectiveId: account.id,
+        },
+      });
+    },
+  },
+  replyToMemberInvitation: {
+    type: GraphQLBoolean,
+    description: 'Endpoint to accept or reject an invitation to become a member',
+    args: {
+      invitation: {
+        type: new GraphQLNonNull(MemberInvitationReferenceInput),
+        description: 'Reference to the invitation',
+      },
+      accept: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+        description: 'Whether this invitation should be accepted or declined',
+      },
+    },
+    async resolve(_, args, req) {
+      if (!req.remoteUser) {
+        throw new Unauthorized();
+      }
+
+      const invitation = await fetchMemberInvitationWithReference(args.invitation, { throwIfMissing: true });
```

Comment:
If we don't add a check for it, anyone will be able to trigger acceptations or denial of invitations on accounts they don't own

```suggestion
      const invitation = await fetchMemberInvitationWithReference(args.invitation, { throwIfMissing: true });
      if (!req.remoteUser.isAdmin(invitation.MemberCollectiveId)) {
        return new Forbidden('Only an admin of the invited account can reply to the invitation');
      }
```